lib/localeapp/cli/install.rb

```diff
@@ -2,35 +2,39 @@ module Localeapp
   module CLI
     class Install < Command
       attr_accessor :config_type
+      attr_accessor :write_dotenv
 
       def initialize(args = {})
         super
         @config_type = :default
+        @write_dotenv = false
       end
 
       def execute(key = nil)
-        installer("#{config_type.to_s.capitalize}Installer").execute(key)
+        installer("#{config_type.to_s.capitalize}Installer").execute(key, write_dotenv)
       end
 
       def installer(installer_class)
         self.class.const_get(installer_class).new(@output)
       end
 
       class DefaultInstaller
-        attr_accessor :key, :project_data, :config_file_path, :data_directory
+        attr_accessor :key, :project_data, :config_file_path, :data_directory, :dotenv
 
         def initialize(output)
           @output = output
         end
 
-        def execute(key = nil)
+        def execute(key = nil, dotenv = false)
           self.key = key
+          self.dotenv = dotenv
```

Comment:
Since you use the `dotenv` argument to mutate state, a better option
would be to give it to the constructor.

But an even better option would be not to mutate state at all, and
remove the `dotenv` instance variable from `DefaultInstaller`. Are you
sure we need/want it?